qa/common/shm_util.py

```diff
@@ -215,3 +221,58 @@ def unregister_cleanup_shm_regions(shm_regions, shm_handles, precreated_shm_regi
                     shm_regions[2+i]+'_data')
                 shm.destroy_shared_memory_region(shm_handles[3])
 
+
+def create_set_either_shm_region(shm_region_names, input_list, input_byte_size,
```

Comment:
Will those helper functions be reused in place other than infer_zero? It seems to be too specific

Reply:
No, they are only used in infer_zero for the L0_infer_zero test

Reply:
Then move them to infer_zero

Reply:
`create_set_shm_regions` and `register_add_shm_regions` are also only used for L0_infer but these helper functions are kept in `shm_util` instead of `inter_util`. Are you suggesting that we get rid of shm_util entirely?

Reply:
Personally I will remove it because the functions in `shm_util` only work for `infer_util`, like what we do in sequence util. But we can leave it as it is and revisit it later, for now, you can merge it if it fixes L0_infer_zero.

Reply:
Will resolve this in a future PR. For now it works and fixes L0_infer_zero for V2